distribution/bwc/build.gradle

```diff
@@ -64,8 +64,12 @@ if (enabled) {
   File checkoutDir = file("${buildDir}/bwc/checkout-${bwcBranch}")
 
   task createClone(type: LoggedExec) {
-    onlyIf { checkoutDir.exists() == false }
-    commandLine = ['git', 'clone', rootDir, checkoutDir]
+    if (checkoutDir.exists() == false) {
+      commandLine = ['git', 'clone', rootDir, checkoutDir]
+    } else {
+      commandLine = ['git', 'fetch', 'origin']
```

Comment:
I don't think this is correct. A clone should always be created. The `origin` remote only comes into play here because you know `origin` is the local checkout the bwc checkout was cloned from.